apps/src/javalab/Playground.js

```diff
@@ -106,13 +135,28 @@ export default class Playground {
       // can't remove items if game is over
       return;
     }
+    if (this.imageItemExists(itemData)) {
+      this.imageItemIds.splice(this.imageItemIds.indexOf(itemData.id), 1);
+      this.removePlaygroundItem(itemData.id);
+    }
+    // TODO: handle text deletion
   }
 
   changeItem(itemData) {
     if (this.isGameOver) {
       // can't change items if game is over
       return;
     }
+    if (this.imageItemExists(itemData)) {
```

Comment:
What if it doesn't exist?

Reply:
for now I'm just ignoring those. An example of when we would want to ignore is if a setter is called on an item that hasn't been added to the playground--right now we still send the message, but we can't do anything with it.

Reply:
👍